airflow/configuration.py

```diff
@@ -142,7 +142,10 @@ def run_command(command):
     },
     'github_enterprise': {
         'api_rev': 'v3'
-    }
+    },
+    'admin': {
+        'hide_confidential_ui_fields': True,
+    },
 }
```

Comment:
Make sure you have specified the config value everywhere it is needed. I recall there are at least 3 places where new config needs to be specified.


For example, look at `dags_are_paused_at_creation` :

```
(venv) Sid-As-MacBook-Pro:airflow siddharth$ grep -ri dags_are_paused_at_creation . | grep -v Binary | grep -v static
./configuration.py:        'dags_are_paused_at_creation': True,
./configuration.py:dags_are_paused_at_creation = True
./configuration.py:dags_are_paused_at_creation = False
./models.py:    is_paused_at_creation = configuration.getboolean('core', 'dags_are_paused_at_creation')
```